AsyncDisplayKit/ASDisplayNode.h

```diff
@@ -15,16 +15,24 @@
 
 #import <AsyncDisplayKit/ASLayoutable.h>
 
+@class ASDisplayNode;
+
 /**
  * UIView creation block. Used to create the backing view of a new display node.
  */
 typedef UIView *(^ASDisplayNodeViewBlock)();
+
 /**
  * CALayer creation block. Used to create the backing layer of a new display node.
  */
 typedef CALayer *(^ASDisplayNodeLayerBlock)();
 
 /**
+ * ASDisplayNode loaded callback block. Used for any additional setup after node's layer/view has been loaded
```

Comment:
Not essential now, but we should 1) mention that this is always called on the main thread, and 2) maybe we should AppleDoc these :)